sensors/sqs_service_sensor.yaml

```diff
@@ -0,0 +1,15 @@
+---
+  class_name: "AWSSQSServiceSensor"
+  entry_point: "sqs_service_sensor.py"
+  description: "Service Sensor which monitors a SQS queue for new messages"
```

Comment:
Would also be good to document here in the description and also in README how this sensor differentiates from other one :)